src/pudl/output/ferc1.py

```diff
@@ -2704,6 +2721,11 @@ def out_ferc1__yearly_rate_base(
     )
     cash_working_capital = prep_cash_working_capital(
         core_ferc1__yearly_operating_expenses_sched320
+    ).merge(
+        core_pudl__assn_ferc1_pudl_utilities,
+        on="utility_id_ferc1",
+        how="left",
+        validate="many_to_one",
```

Comment:
Necessary because this table doesn't go through the same explosion process as all the others, which now adds the utility name and ID columns.